ui/src/store/link-opener-store.ts

```diff
@@ -0,0 +1,68 @@
+import { makeAutoObservable } from 'mobx'
+
+import { serviceLocator } from '@/services/service-locator'
+
+import { DeviceControlStore } from './device-control-store'
+import { deviceBySerialStore } from './device-by-serial-store'
+
+export class LinkOpenerStore {
+  currentBrowserId: string | undefined
+
+  private readonly deviceControlStore: DeviceControlStore
+
+  constructor(serial: string) {
+    makeAutoObservable(this)
+
+    this.deviceControlStore = serviceLocator.get<DeviceControlStore>(DeviceControlStore.name)
+
+    this.init(serial)
+  }
+
+  async init(serial: string): Promise<void> {
```

Comment:
I would love if the last selected browser would be persisted in the local storage